src/BuildingRegistry.Api.CrabImport.Handlers.Sqs.Lambda/.lambda-test-tool/SavedRequests/CrabImport.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Shouldn't this be under the test project?

Reply:
shouldn't be there at all! will remove